src/commands/Games/wheeloffortune.ts

```diff
@@ -0,0 +1,48 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Emojis } from '../../lib/util/constants';
+import { WheelOfFortune } from '../../lib/util/Games/WheelOfFortune';
+import { getColor } from '../../lib/util/util';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['wof'],
+			cooldown: 10,
+			description: language => language.tget('COMMAND_WHEELOFFORTUNE_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_WHEELOFFORTUNE_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [text]: [string]) {
+		await WheelOfFortune.init();
```

Comment:
This doesn't have to be called every message, does it now? If not, please move to the init function

Reply:
In fact, you init it in klasaReady...why call it again?

Reply:
woops leftover from testing